docs/api-reference/leaksdb/endpoints/post-credentials-search.mdx

```diff
@@ -86,6 +86,16 @@ See the guide for using this endpoint:
     ```
     </Tab>
 
+    <Tab title="Auth Domain Query" >
+    This query will match the domain of the url that this credential might have been used to log in to.
```

Comment:
Also if you don't mind I might want to nitpick the wording here...

You don't really log into an url, you log into a service.

So maybe something like:
- `This query will match the domain of the service that this credential might have been used to log in to.`